### PR TITLE
pgsodium/vault smoke tests

### PR DESCRIPTION
**Pull request by michelp**
## What kind of change does this PR introduce?

Add smoke tests for pgsodium/vault

### CODE REVIEW

**Comment by michelp** [comment]
I meant to make this a draft don't merge this yet.

File: tests/smoke/0002-pgsodium-vault.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
+BEGIN;
+
+CREATE EXTENSION IF NOT EXISTS pgtap;
+CREATE EXTENSION IF NOT EXISTS supabase_vault CASCADE;
```
Comment (thoughtpolice) [inline tests/smoke/0002-pgsodium-vault.sql:4]: Shouldn't need these, FWIW

**Comment by steve-chavez** [comment]
@michelp Sorry, you might need to rebase bc https://github.com/supabase/nix-postgres/pull/4/ was merged and contains a `0002-supautils.sql`.

@thoughtpolice Does the numbering in the sql files matter?

**Comment by thoughtpolice** [comment]
@steve-chavez No, actually I used that because I figured to avoid some conflicts people could choose random increments, the same way people would number lines in BASIC as `100` or `200` instead of `1` and `2`, allowing more lines to be added in the middle later. :)

**Pull request closed without merging**